lib/configs/recommended.js

```diff
@@ -0,0 +1,104 @@
+    'brace-style': ['error', '1tbs', { 'allowSingleLine': true }],
+    'comma-dangle': ['error', 'always-multiline'],
+    'comma-spacing': 'error',
+    'comma-style': 'error',
+    'computed-property-spacing': 'error',
+    'constructor-super': 'error',
+    'curly': 'error',
+    'eol-last': 'error',
+    'eqeqeq': 'error',
+    'func-call-spacing': 'error',
+    'func-style': ['error', 'declaration', { 'allowArrowFunctions': true }],
+    'indent': ['error', 2],
+    'jsx-quotes': ['error', 'prefer-double'],
+    'key-spacing': ['error', { 'beforeColon': false, 'afterColon': true }],
+    'keyword-spacing': 'error',
+    'linebreak-style': 'error',
+    'new-parens': 'error',
+    'no-extra-bind': 'error',
+    'no-extra-parens': ['error', 'all', { 'ignoreJSX': 'multi-line' }],
+    'no-lonely-if': 'error',
+    'no-mixed-spaces-and-tabs': 'error',
+    'no-multi-spaces': 'error',
+    'no-multiple-empty-lines': ['error', { 'max': 1 }],
+    'no-shadow': 'error',
+    'no-tabs': 'error',
+    'no-trailing-spaces': 'error',
+    'no-unneeded-ternary': 'error',
+    'no-unsafe-negation': 'error',
+    'no-useless-return': 'error',
+    'no-var': 'error',
+    'no-whitespace-before-property': 'error',
+    'object-curly-newline': ['error', {
+      'ObjectExpression': { 'minProperties': 1, 'multiline': true },
+      'ObjectPattern': { 'minProperties': 3, 'multiline': true }
+    }],
+    'object-curly-spacing': ['error', 'always', { 'arraysInObjects': false }],
+    'object-property-newline': 'error',
+    'object-shorthand': 'error',
+    'operator-assignment': 'error',
+    'padded-blocks': ['error', 'never'],
+    'prefer-const': 'error',
+    'quotes': ['error', 'single'],
+    'react/display-name': 'off',
+    'react/jsx-closing-bracket-location': 'error',
+    'react/jsx-closing-tag-location': 'error',
+    'react/jsx-curly-spacing': ['error', { 'when': 'never', 'children': true }],
+    'react/jsx-equals-spacing': ['error', 'never'],
+    'react/jsx-first-prop-new-line': ['error', 'multiline'],
+    'react/jsx-indent-props': ['error', 2],
+    'react/jsx-indent': ['error', 2],
+    'react/jsx-max-props-per-line': ['error', { 'maximum': 1 }],
+    'react/no-deprecated': ['error'],
+    'react/jsx-tag-spacing': ['error', { 'closingSlash': 'never', 'beforeSelfClosing': 'always', 'afterOpening': 'never' }],
+    'react/jsx-wrap-multilines': ['error'],
+    'react/self-closing-comp': 'error',
+    'react/sort-prop-types': ['error', { 'ignoreCase': true }],
+    'rest-spread-spacing': 'error',
+    'root/integration-drivers-return-this-or-null': 'error',
+    'root/integration-driver-import': 'error',
+    'root/integration-test-format': 'error',
+    'root/prefer-root-text': 'error',
+    'root/prevent-use-of-window-location': 'error',
+    'root/preceded-by-await': ['error', { 'functionNames': ['pollForCondition', 'wait', 'waitForElement'] }],
+    'root/sort-reducers-index': 'error',
+    'semi-spacing': 'error',
+    'semi': 'error',
```

Comment:
I would love to see this rule go away 😅

Reply:
Luckily, I am one of those people that loves semicolons and that was a previously enforced rule across projects. :)